sds_data_manager/lambda_code/SDSCode/dependency_config.py

```diff
@@ -801,7 +801,6 @@
 
 # Get dependencies for pre-processing dependency table
 for dep in downstream_dependents:
-    if dep.reverse_direction:
-        upstream_dependents.append(dep.reverse_direction())
+    upstream_dependents.append(dep.reverse_direction())
```

Comment:
Minor nit, but you could write this as a one-line list comprehension as well.
```suggestion
    upstream_dependents = [dep.reverse_direction() for dep in downstream_dependents]
```

Reply:
for sure!